### ISSUE TITLE
Enabling logging on Android crashes the app

### ISSUE THREAD

**Issue by matejdro** [facebook/yoga#1120]
# Report

# Issues and Steps to Reproduce

1. Create a YogaConfig with a logger
2. Create a YogaNode with that config
3. Attempt to layout it

# Expected Behavior

Logger should work

# Actual Behavior
Whole app crashes with

```
signal 6 (SIGABRT), code -1 (SI_QUEUE), fault addr --------
Abort message: 'JNI DETECTED ERROR IN APPLICATION: illegal class name 'Lcom/facebook/yoga/YogaLogLevel;'
        (should be of the form 'package/Class', [Lpackage/Class;' or '[[B')
        in call to FindClass
        from void com.facebook.ckage/Class', [Lpackage/Class;' or '[[B')
        in call to FindClass
        from void com.facebook.yoga.YogaNative.jni_YGNodeCalculateLayoutJNI(long, float, float, long[], com.facebook.yoga.YogaNodeJNIBase[])'
    x0  0000000000000000  x1  0000000000002c71  x2  0000000000000006  x3  0000007fdd736130
    x4  fefeff7edc724f04  x5  fefeff7edc724f04  x6  fefeff7edc724f04  x7  7f7f7f7fff7f7f7f
    x8  00000000000000f0  x9  0000007e4e49a118  x10 0000000000000000  x11 ffffff80fffffbdf
    x12 0000000000000001  x13 0000000019c11820  x14 0000007fdd734f40  x15 ffffffffffffffff
    x16 0000007e4e56fd20  x17 0000007e4e54a130  x18 0000007e5cace000  x19 0000000000002c71
    x20 0000000000002c71  x21 00000000ffffffff  x22 0000000000000000  x23 0000000000000000
    x24 0000007dba8b5ed6  x25 0000000000000001  x26 0000000000000000  x27 0000007dbb217000
    x28 b400007e5c3076c0  x29 0000007fdd7361b0
    lr  0000007e4e4fb938  sp  0000007fdd736110  pc  0000007e4e4fb968  pst 0000000000000000
```

**Comment by matejdro** [comment]
@evancharlton this bug was not react-related though (we are not using React, only Yoga). Does this fix in another repository also address native Yoga?

**Comment by evancharlton** [comment]
My apologies - this shouldn't have been closed. I originally patched it with #1156 , but then cherry-picked it to react-native (where I experienced it) because they have copied yoga into react-native directly (possibly because of #1151 ?). When I did that, it copied over the issue attribution, and that closed the issue.

Hopefully someone can reopen it (and/or review #1156)
